__tests__/transformers/reusable-content.test.js

```diff
@@ -0,0 +1,45 @@
+import { mdast } from '../../index';
+
+describe('reusable content transfomer', () => {
+  it('should replace a reusable content block if the block is provided', () => {
+    const reusableContent = {
+      test: `
+# Test
+
+[link](http://example.com)
+    `,
+    };
+    const md = `
+Before
+
+<RMReusableContent name="test" />
+
+After
+    `;
+
+    const tree = mdast(md, { reusableContent });
+
+    expect(tree.children[0].children[0].value).toBe('Before');
+    expect(tree.children[1]).toMatchSnapshot();
+    expect(tree.children[2].children[0].value).toBe('After');
+  });
+
+  it('should parse a reusable content block that is not defined', () => {
```

Comment:
```suggestion
  it('should not parse a reusable content block that is not defined', () => {
```
If I'm reading this test case correctly, it doesn't get parsed right?

Reply:
Err, I think it's still getting parsed, per se. It's just empty? Maybe it could be clearer with "should insert an empty node if the reusable content block is not defined"

Reply:
Ah yeah that makes more sense 👍